### PR TITLE
1.1/512 tests for add voucher batches

### PR DESCRIPTION
**Pull request by ob6160**

### CODE REVIEW

**Comment by ob6160** [comment]
@charlesstrange2 I'm tempted just to extend "testStoreBatch" which already checks that vouchers were created so that it also checks that they are within the correct code range. Thoughts?

**Comment by charlesstrange2** [comment]
I think the test should be distinct. "testStoreBatch" currently checks the UI response to the adding of vouchers. Rename it to "testStoreBatchSuccessMsg" or similar, then clone the vocher adding part into a new "testStoreBatch" test and continue from there.

File: tests/Unit/Controllers/Service/Admin/VoucherControllerTest.php [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        // Assert that eleven vouchers have been made and are in the expected state and land within the expected range.
+        $this->assertCount(11, $vouchers);
+        foreach($vouchers as $voucher) {
+            $this->assertEquals('printed', $voucher->currentstate);
+
+            // Assert that the voucher code starts with the Sponsor shortcode.
+            $this->assertStringStartsWith($shortcode, $voucher->code);
+
+            // Assert that the voucher code lands between 50 and 60 (our generated range).
+            $this->assertRegExp('/,*[5-6][0-9]/', $voucher->code);
```
Comment (charlesstrange2) [inline tests/Unit/Controllers/Service/Admin/VoucherControllerTest.php]: technically this matches 50-69; can you narrow it?

Comment (ob6160) [inline tests/Unit/Controllers/Service/Admin/VoucherControllerTest.php]: I'll look into narrowing after I've got the fix in for the number of generated vouchers in that other ticket.

**Comment by charlesstrange2** [comment]
Yup, that makes sense

**Pull request merged**